trieste/models/gpflux/interface.py

```diff
@@ -20,11 +20,11 @@
 from gpflow.base import Module
 
 from ...types import TensorType
-from ..interfaces import ProbabilisticModel
+from ..interfaces import SupportsGetObservationNoise
 from ..optimizer import BatchOptimizer
 
 
-class GPfluxPredictor(ProbabilisticModel, ABC):
+class GPfluxPredictor(SupportsGetObservationNoise, ABC):
```

Comment:
should be get_kernel as well and few other things no?

Reply:
ABC should be removed here no? (you did so for GPflowPredictor)

Reply:
Good spot. Might actually put ABC back in GPflowPredictor and KerasPredictor: it makes no practical difference (except to allow you to register a non-subclass as an "implementation") but makes it clearer that these are neither concrete model implementations nor protocols.